cmd/main_test.go

```diff
@@ -259,6 +498,33 @@ func prepareTestConfig() *cfg.Config {
 		SourceQuery:          "select * from mydb.test_table",
 		SourceSplitKey:       "id",
 		SourceSplitTimeKey:   "",
+		DatabendDSN:          "https://dbarchiver:abc123@tn3ftqihs--medium-p8at.gw.aws-us-east-2.default.databend.com:443",
```

Comment:
Be care of the `DatabendDSN`, maybe change it to `http://databend:databend@localhost:8080` is better

Reply:
My mistake. Forget to remove mysql config.